amaznot/database/seeders/ProductSeeder.php

```diff
@@ -32,20 +32,27 @@ public function run()
 
                     $categories = explode("|", $data[1]);
                     $categories = str_replace('&', 'and', $categories);
-                    
+                    $about = str_replace('|', ' ', $data[3]);
+                    $details = str_replace('|', ' ', $data[4]);
```

Comment:
Please revert the changes for the seeding logic of the about and details columns. If you wish to replace these | characters with spaces for display, it should be handled in the view.